GetSubscriptionsFeed/utils.ts

```diff
@@ -7,7 +7,7 @@ import { FiscalCodeHash } from "../generated/definitions/FiscalCodeHash";
 /**
  * A minimal storage table Entry
  */
-type TableEntry = Readonly<{
+export type TableEntry = Readonly<{
```

Comment:
there's an open PR to add these types into io-functions-commons here:  https://github.com/teamdigitale/io-functions-commons/pull/31/files#diff-6acba14f29206bf08111645a46659d1fR233

cc @francescopersico

Reply:
The PR has been splitted in 2 parts and is already in master. The part that introduce this type is here https://github.com/teamdigitale/io-functions-commons/pull/34/files#diff-6acba14f29206bf08111645a46659d1fR233.

Reply:
ok i will import the correct type